src/main/scala/gitbucket/core/controller/api/ApiRepositoryContentsControllerBase.scala

```diff
@@ -34,22 +37,32 @@ trait ApiRepositoryContentsControllerBase extends ControllerBase {
     getContents(repository, multiParams("splat").head, params.getOrElse("ref", repository.repository.defaultBranch))
   })
 
-  private def getContents(repository: RepositoryService.RepositoryInfo, path: String, refStr: String) = {
-    def getFileInfo(git: Git, revision: String, pathStr: String): Option[FileInfo] = {
+  private def getContents(
+    repository: RepositoryService.RepositoryInfo,
+    path: String,
+    refStr: String,
+    matchByLowCase: Boolean = false
+  ) = {
+    def getFileInfo(git: Git, revision: String, pathStr: String, matchByLowCase: Boolean): Option[FileInfo] = {
```

Comment:
IMHO, `ignorecase` would be better.

Reply:
yeah... true... I fixed arg name in https://github.com/gitbucket/gitbucket/pull/2535/commits/f346ad9b55ea83eb93f4a25b43b67fa422949d42
Thanks!